### PR TITLE
fix: connecting with wallet connect on mobile

### PR DESCRIPTION
**Pull request by blushi**
## Description

https://regennetwork.atlassian.net/browse/APP-115?atlOrigin=eyJpIjoiYzY2MjE2ZjdkYThlNGU3M2E5NTg2MDMxZjgyNDBmM2QiLCJwIjoiaiJ9

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

Try to log in via wallet connect on mobile from https://deploy-preview-2377--regen-marketplace.netlify.app/
Once connected, reload the page, it should not show the prompt to open the keplr app again.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-marketplace/package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "@cosmjs/cosmwasm-stargate": "^0.31.0",
     "@cosmjs/launchpad": "^0.27.0",
     "@cosmjs/stargate": "^0.27.0",
-    "@cosmos-kit/keplr-mobile": "^2.6.7",
-    "@cosmos-kit/react-lite": "^2.6.7",
+    "@cosmos-kit/keplr-mobile": "^2.9.0",
+    "@cosmos-kit/react-lite": "^2.9.0",
```
Comment (blushi) [inline web-marketplace/package.json:14]: this wasn't needed to fix the issue but while debugging, I upgraded those packages that we use to connect via wallet connect

File: web-marketplace/src/lib/wallet/wallet.tsx [hunk 2]
```diff
@@ -110,16 +108,14 @@ export const WalletProvider: React.FC<React.PropsWithChildren<unknown>> = ({
 
   // Connecting via Wallet Connect is handled entirely using @cosmos-kit
   const [walletConnect, setWalletConnect] = useState<boolean>(false);
-  const { client: walletConnectClient } = useWalletClient(KEPLR_MOBILE);
+  const { client: walletConnectClient, status } = useWalletClient(KEPLR_MOBILE);
   const { mainWallet } = useCosmosKitWallet(KEPLR_MOBILE);
-  const { walletRepos } = useManager();
-  const { closeView } = useChain('regen');
+  const { closeView } = useChain('regen', false);
```
Comment (blushi) [inline web-marketplace/src/lib/wallet/wallet.tsx:113]: setting `sync` (second argument) to false fixed the issue

**Review by aaronc — APPROVED** [review]
utACK

**Comment by erikalogie** [comment]
LGTM

**Pull request merged**